app/build.gradle

```diff
@@ -47,12 +47,12 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "com.android.support:appcompat-v7:$support"
-    implementation "com.android.support:design:$support"
-    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+    implementation 'androidx.appcompat:appcompat:1.0.0'
```

Comment:
pkoi ne pas garder la variable $support et mettre =1.0.0

Reply:
Oui tu as bien raison. Du coup c'est le boutton Migrate to AndroidX sur Android Studio qui m'a fait cela. Je fixe